src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.manual.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+[assembly:System.Runtime.CompilerServices.InternalsVisibleTo("mscorlib, PublicKey=00000000000000000400000000000000")]
+
+namespace System.Runtime.CompilerServices
+{
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHere
+    {
+        internal AssemblyAttributesGoHere()
+        {
+        }
+    }
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHereS
```

Comment:
Gotta love these names. :)

Reply:
Yeah, that's why we're leaving them internal rather than making public.